src/models/artefact.ts

```diff
@@ -25,14 +25,24 @@ export class Artefact {
     constructor(obj: ArtefactDTO) {
         this.id = obj.id;
         this.editionId = obj.editionId;
-        this.imagedObjectId = obj.imagedObjectId;
+        this.imagedObjectId = obj.imagedObjectId || '';
         this.name = obj.name;
         this.mask = Polygon.fromWkt(obj.mask);
         this.artefactMaskEditorId = obj.artefactMaskEditorId;
         this.isPlaced = obj.isPlaced;
         this.placement = new Placement(obj.placement);
         this.artefactPlacementEditorId = obj.artefactPlacementEditorId;
         this.side = (obj.side === 'recto') ? 'recto' : 'verso';
+
+        // TODO: Check if the artefact ID is on of the list of "virtual" artefacts. If so, set their
+        // imageObjectId to '', so that they are treated as virtual
+        if (this.id === 16772) {
```

Comment:
This doesn't belong in the PR, does it?  It looks like some debugging stuff.

Reply:
Oh, right! We should take this out!